### PR TITLE
Fixing Glottocode

### PR DESCRIPTION
**Pull request by CMUgarte**
@FredericBlum, here is the fix for [lexibank/lexibank-analysed/issues/35](https://github.com/lexibank/lexibank-analysed/issues/35) and [lexibank/lexibank-analysed/issues/47](https://github.com/lexibank/lexibank-analysed/issues/47).

### CODE REVIEW

**Comment by MuffinLinwist** [comment]
@FredericBlum, this is ready for your review.

**Comment by CMUgarte** [comment]
@FredericBlum, this is ready for your review.

**Comment by CMUgarte** [comment]
@chrzyki, I will also need write access for this repo :)

**Comment by chrzyki** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CLDF Dataset derived from Saenko's "Annotated Swadesh wordlists for the Romance group" from 2015
 
-[![CLDF validation](https://github.com/lexibank/saenkoromance/workflows/CLDF-validation/badge.svg)](https://github.com/lexibank/saenkoromance/actions?query=workflow%3ACLDF-validation)
+[![CLDF validation](https://github.com/MuffinLinwist/saenkoromance/workflows/CLDF-validation/badge.svg)](https://github.com/MuffinLinwist/saenkoromance/actions?query=workflow%3ACLDF-validation)
```
Comment (FredericBlum) [inline README.md:3]: In this PR the links are not yet fixed

**Review by FredericBlum — COMMENTED** [review]
Glottocode is good, but I think the CLDF validation needs changes, right?

**Comment by CMUgarte** [comment]
> Glottocode is good, but I think the CLDF validation needs changes, right?

yes, I'm only doing it now that I have the write access :)

**Pull request closed without merging**